### PR TITLE
feat: extend metrics

### PR DESCRIPTION
**Pull request by cv65kr**
# Reason for This PR

https://github.com/roadrunner-server/roadrunner/issues/1194

## Description of Changes

Added metrics - Latency, RPM

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the MIT license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

### CODE REVIEW

File: metrics.go [hunk 1]
```diff
@@ -53,6 +55,17 @@ func newStatsExporter(stats Informer, jobsOk, pushOk, jobsErr, pushErr *uint64)
 		pushErrDesc: prometheus.NewDesc(prometheus.BuildFQName(namespace, "", "push_err"), "Number of jobs push which was failed", nil, nil),
 		jobsErrDesc: prometheus.NewDesc(prometheus.BuildFQName(namespace, "", "jobs_err"), "Number of jobs error while processing in the worker", nil, nil),
 		jobsOkDesc:  prometheus.NewDesc(prometheus.BuildFQName(namespace, "", "jobs_ok"), "Number of successfully processed jobs", nil, nil),
+
+		pushJobLatencyHistogram: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Name: prometheus.BuildFQName(namespace, "", "push_latency"),
+			Help: "Histogram represent latency for pushed operation",
```
Comment (rustatian) [inline metrics.go:61]:
```suggestion
			Help: "Histogram represents latency for pushed operation",
```

File: metrics.go [hunk 2]
```diff
@@ -53,6 +55,17 @@ func newStatsExporter(stats Informer, jobsOk, pushOk, jobsErr, pushErr *uint64)
...
+		}, []string{"job", "driver", "source"}),
+
+		pushJobRequestCounter: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "push_request_total",
```
Comment (rustatian) [inline metrics.go:66]:
```suggestion
			Name:      "requests_total",
```

File: metrics.go [hunk 3]
```diff
@@ -53,6 +55,17 @@ func newStatsExporter(stats Informer, jobsOk, pushOk, jobsErr, pushErr *uint64)
...
+			Help:      "Total number of pushed requests",
```
Comment (rustatian) [inline metrics.go:67]:
```suggestion
			Help:      "The total number of requests sent to the JOBS plugin",
```

File: plugin.go [hunk 4]
```diff
@@ -363,6 +367,7 @@ func (p *Plugin) PushBatch(j []jobsApi.Job) error {
 	start := time.Now()
 
 	for i := 0; i < len(j); i++ {
+		operationStart := time.Now()
```
Comment (rustatian) [inline plugin.go:370]:
```suggestion
		operationStart := time.Now().UTC()
```

**Comment by cv65kr** [comment]
@rustatian all updated.

**Review by rustatian — APPROVED** [review]
LGTM

**Comment by rustatian** [comment]
Thanks @cv65kr ❤️

**Pull request merged**